website/client/src/components/chat/chatCard.vue

```diff
@@ -34,6 +34,8 @@
       <div
         ref="markdownContainer"
         class="text markdown"
+        :class="'text-' + languageAlign"
+        :style="'direction:' + languageDirection"
```

Comment:
Can this be changed to use an object syntax? see https://vuejs.org/v2/guide/class-and-style.html#Object-Syntax-1

Reply:
done